### PR TITLE
[Website] add lyft case-study

### PR DESCRIPTION
**Pull request by bullet03**
NB!!! Please, merge https://github.com/apache/beam/pull/21908 first, before merging current PR
- add lyft.md case-study
- add related images
- change _case-study.scss
------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by bullet03** [comment]
R: @nausharipov

File: website/www/site/content/en/case-studies/hop.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ cardDescription: "Apache Hop is an open source data orchestration and engineerin
 authorName: "Matt Casters"
 authorPosition: "Chief Solutions Architect, Neo4j, Apache Hop co-founder"
 authorImg: /images/matt_casters_photo.png
+date: 2022-05-01T07:00:00+00:00
```
Comment (AKosolapov) [inline website/www/site/content/en/case-studies/hop.md:11]: Can we use commit dates from histrory as publish dates to resemble ordering? I would also ask to rename to publishDate to best describe and map to Hugo formatting.

Comment (bullet03) [reply website/www/site/content/en/case-studies/hop.md]: done, separate PR created, renamed the value https://github.com/apache/beam/pull/21908

**Comment by AKosolapov** [comment]
retest this please

**Comment by AKosolapov** [comment]
retest this please

**Comment by AKosolapov** [comment]
Looks good to me

**Comment by pabloem** [comment]
LGTM. Happy to merge if other reviews are ready?

**Comment by aijamalnk** [comment]
LGTM. Thank you all for a great story :)

**Pull request merged**